pkg/collector/corechecks/ebpf/probe/tcp_queue_length.go

```diff
@@ -145,3 +155,35 @@ func (t *TCPQueueLengthTracer) GetAndFlush() TCPQueueLengthStats {
 
 	return result
 }
+
+func loadTCPQueueLengthCOREProbe(cfg *ebpf.Config) (*TCPQueueLengthTracer, error) {
+	kv, err := kernel.HostVersion()
+	if err != nil {
+		return nil, fmt.Errorf("error detecting kernel version: %s", err)
+	}
+	if kv < kernel.VersionCode(4, 9, 0) {
+		return nil, fmt.Errorf("detected kernel version %s, but tcp-queue-length probe requires a kernel version of at least 4.8.0", kv)
```

Comment:
It looks like these aren't matching (4.9.0 version code check vs. 4.8.0 in the log)

Reply:
Good catch. Copy-pasted code from OOM Kill which does required 4.9